account/models.py

```diff
@@ -9,7 +9,7 @@ class Account(AbstractUser):
     submit_time = models.IntegerField(default=0)
 
     list_column = models.IntegerField(default=20)
-
+    nav_color = models.TextField(default="#545c64")
```

Comment:
You should not fillup default value like magic everywhere, I suggest:

```suggestion
    nav_color = models.TextField(blank=True,null=True)
```

And then in the frontend:

```javascript
let nav_color = '#545c64';
if (account_info.nav_color != null) { nav_color = account_info.nav_color; }
```